<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editComputer</class>
 <widget class="QDialog" name="editComputer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="C_input_imagepath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="InputComputerName">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>291</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="InvalidInput">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Invalid input</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="TypeDropDown">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Mechanical</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Vacuum</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Transistors</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Digital</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Other...</string>
    </property>
   </item>
  </widget>
  <widget class="QRadioButton" name="RadioWasBuilt">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Was built?</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="InputBuiltYear">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>120</y>
     <width>93</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Build year</string>
   </property>
  </widget>
  <widget class="QPushButton" name="C_add_ok">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>93</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="InputComputerType">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>93</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="c_showUrl">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Add new computer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addWebpage">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add webpage...</string>
   </property>
  </widget>
  <widget class="QLabel" name="ComputerImageLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>161</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/icons/noImage.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="browseImagePath">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>210</y>
     <width>93</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
